### PR TITLE
Prevent mixed content error by including fonts directly

### PR DESCRIPTION
**Pull request by tomayac**
Even if one references https://db.onlinewebfonts.com/c/88bdc70ec292cc29451932f9875b378a?family=SegoeUIMonoW01-Regular explicitly over https, the resulting font URLs still use http. Sad. This pull request fixes this issue.
CC: @boyofgreen

### CODE REVIEW

**Review by nmetulev — APPROVED** [review]

**Comment by nmetulev** [comment]
@tomayac, you need to sign the contributor license agreement before we can merge

**Comment by tomayac** [comment]
Thanks. I’m about to get approval for signing the CLA. Hope this gets sorted soon.

**Comment by tomayac** [comment]
Turns out it's not easy to get approval to sign the CLA, because reasons 🤷. I fear I have to close this PR, but maybe you can still apply the changes independent from it?

**Pull request closed without merging**
